File: Strings/silly_sentences.c
//Tatiana Susov, Silly Sentences, C
#include <stdio.h>
//String variables for my user inputs (minimum of 3)
 char animal [943287];
 char verb [57984203];
 char place [5793454];
 char feeling [57439857];
 char noun [9852340];
 
int main(void){
   
 printf("Hello user and welcome to the crazy fantastic amazing bizarre mad lib program, lets make a mad lib together!");
 printf("\nPlease input a random animal (single word).");
 scanf("%s", animal );
 printf("\nNow input a past tense verb (single word) ");
 scanf("%s", verb );
 printf("\nNow, input a place of your choice (single word) ");
 scanf("%s", place );
 printf("\nNow input a feeling (single word). ");
 scanf("%s", feeling);
 printf("\nLast but not least, please input a noun (single word). ");
 scanf("%s", noun );
 printf("\nHere is your mad lib! "); 
 printf("\n The ");
 printf("%s", animal );
 printf(" ");
 printf("%s",verb );
 printf(" all the way to ");
 printf("%s", place);
 printf(" with a/an ");
 printf("%s", feeling);
 printf(" look on his face. Don't worry, he didn't forget his ");
 printf("%s", noun);
 printf(".");

    return 0;
}

File: Strings/silly_sentences.py
# Tatiana Susov, Silly Sentences Python
#name = input ("What is your first name?\n")


print ("Welcome to the silly sentences program, a program designed to make a silly mad-lib. Let's begin!\n")
animal = input ("Please input the name of an animal\n").strip()
noun_1 = input ("Please input a noun of your choice\n").strip()
verb_pt = input ("Please input a present tense verb\n").strip()
noun_2 = input ("Please input a second noun of your choice\n").strip()
place = input ("Please input a place of your choice").strip()
print ("Here is your mad-lib!")
print ("My", animal,"always takes my", noun_1, "when I am", verb_pt+". So now I have to chase him around with a/an", noun_2, "in my hand so I can get my", noun_1, "back before going to", place+ ".") 
